src/MakePotCommand.php

```diff
@@ -198,6 +202,9 @@ class MakePotCommand extends WP_CLI_Command {
 	 * [--skip-php]
 	 * : Skips PHP string extraction.
 	 *
+	 * [--skip-json]
```

Comment:
Is `--skip-json` the best flag name here? This might be misleading with regards to extracting _into_ a JSON file.

Also, this doesn't control a `JsonExtractor`, but rather a `BlockExtractor`.

I think `--skip-json` should either control extraction from _any_  JSON file, or it should be renamed to what it actually controls, like `--skip-blocks`.

However, `--skip-blocks` would deviate from the "filetype of input file" approach we currently have for this.

Reply:
Yeah this is a bit inconsistent because I renamed the extractor mid-flight. Maybe `skip-block-json` to make it more explicit?

Reply:
Yes, would be less misleading at least.

Reply:
Done in b3c9bd711778daef514bd395746ba7f1349e80c1